apidoc/Titanium/Contacts/Person.yml

```diff
@@ -160,15 +178,31 @@ properties:
         `string` type value.
         
         The `service` key value may be one of `AIM`, `Facebook`, `GaduGadu`, `GoogleTalk`, `ICQ`, 
-        `MSN`, `QQ`, `Skype`, or `Yahoo`.
+        `Jabber`, `MSN`, `QQ`, `Skype`, or `Yahoo`.
```

Comment:
Added support for Jabber in Android in next commit. The keys `home`, `work`, `others` were there before this PR, and if changed, will break the parity for android. The predefined values are actually exactly these `service` key values written in the doc, and are case sensitive.